worker/changestream/eventmultiplexer/eventmultiplexer.go

```diff
@@ -191,6 +206,9 @@ func (e *EventMultiplexer) loop() error {
 		}
 		e.subscriptions = nil
 		e.subscriptionsByNS = nil
+
+		// Clear subscriptions metrics.
+		e.metrics.SubscriptionsClear()
```

Comment:
We don't need to call this one out. Either have a for loop that decrements them all or just ignore.

Reply:
I don't think `DecBy(something)` is a good idea, because we are already decrementing when unsubscribing so if we also decrement with a non-updated value then the gauge will be completely false. I added this case only to clear the gauge up in case of errors on the worker and just to be sure. Do you think I should remove it?

Reply:
> Do you think I should remove it?

Yeah, it serves no purpose.